test/cmd/admin.sh

```diff
@@ -333,7 +333,7 @@ os::cmd::expect_success "oc adm registry --daemonset --images='${USE_IMAGES}'"
 os::cmd::expect_success_and_text 'oc adm registry --daemonset' 'service exists'
 os::cmd::try_until_text 'oc get ds/docker-registry --template="{{.status.desiredNumberScheduled}}"' '1'
 # clean up so we can test non-daemonset
-os::cmd::expect_success "oc adm registry --daemonset -o yaml | oc delete -f -"
+os::cmd::expect_success "oc adm registry --daemonset -o yaml | oc delete -f - -ncmd-admin --cascade=false"
```

Comment:
Anybody signed this off or do we have a follow-up? Looks suspicious. @tnozicka maybe?

Reply:
> Anybody signed this off or do we have a follow-up? Looks suspicious. @tnozicka maybe?

I chased it down to a change in the way pods scheduled to nodes are cleaned up and we have a fake node.  The point of this test wasn't checking deletion, it was to scrub resources for another create.

Reply:
GC should handle cleaning all this, so this lgtm.